gems/aws-sigv4/lib/aws-sigv4/asymmetric_credentials.rb

```diff
@@ -11,8 +11,6 @@ module AsymmetricCredentials
 
       N_MINUS_2 = 0xFFFFFFFF00000000FFFFFFFFFFFFFFFFBCE6FAADA7179E84F3B9CAC2FC632551 - 2
 
-      # @param [String] :access_key_id
```

Comment:
Why remove these?  They are still the correct paramters here?

Reply:
The module is API private and these were wrong. These are positional parameters but they were symbolized name, so it produced a warning. I just deleted them because it's clear what they are (and API private).